api/python/provisioner/srv/salt/repos/upgrade.sls

```diff
@@ -21,11 +21,13 @@
 
 {% for version, repos in pillar['release']['upgrade']['repos'].items() %}
 
-    {% set single_iso_repo = {version: repos.pop(version)} %}
+    {% if not repos.startswith(('http://', 'https://')) %}
```

Comment:
`repos` here is a dictionary but not a string, I guess the better way is to check whether that dictionary has `version` as one of the keys: if [true then it's an ISO](https://github.com/Seagate/cortx-prvsnr/blob/e447eb4f00aaeb955838abe77d3d2d7ae0e5917a/api/python/provisioner/inputs.py#L1140).

Reply:
Done